test/extended/etcd/cert_rotation.go

```diff
@@ -0,0 +1,136 @@
+package etcd
+
+import (
+	"context"
+	g "github.com/onsi/ginkgo/v2"
+	o "github.com/onsi/gomega"
+	"github.com/openshift/library-go/test/library"
+	exutil "github.com/openshift/origin/test/extended/util"
+	"github.com/pkg/errors"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"math/rand"
+	"strings"
+	"time"
+)
+
+var _ = g.Describe("[sig-etcd][Feature:CertRotation] etcd", func() {
+	defer g.GinkgoRecover()
+
+	ctx := context.TODO()
+	oc := exutil.NewCLIWithoutNamespace("etcd-certs").AsAdmin()
+
+	g.It("can manually rotate signer certificates [Slow][Timeout:60m]", func() {
```

Comment:
The tests themselves look good to me 👍 

Although I don't see a run of them in CI. Maybe if we add them to the serial suite? `[Suite:openshift/conformance/serial]`.

IIUC the rotation shouldn't be disruptive, although the extra revision rollouts might trigger some disruption test. If that's the case then maybe it's better off in it's own suite like the recovery and scaling ones.

Reply:
just added it, let's see how the invariants behave :) 
@soltysh any opinion on where to plug these kinds of tests?

Reply:
I'm inclined to say that based on `[Timeout:60m]` and `[Timeout:30m]` on these tests, I'm inclined to say that serial is not a good fit for this test. Probably worth syncing with the rest of the control plane where these kind of tests would be best kept. Seems like having entirely new suite. Alternatively, I'd say this could be part of the `openshift/etcd/recovery` one.

Reply:
reduced the timeouts, for the suite we have to go checkout what happened to the disruptive suite, but I think it was removed entirely with 4.14